### DISCUSSION TITLE
Shader being STUPID

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#454] [Q&A] [answered]
no joke, I literally just copy-and-pasted the default shader script from the wiki into a file and got 6 errors

![Screenshot 2024-11-24 075303](https://github.com/user-attachments/assets/27d4c4aa-a184-45d8-8bbd-e5c40a978d02)
![Screenshot 2024-11-24 075307](https://github.com/user-attachments/assets/1a4a1d2f-9e6f-4be3-a6ac-75626fb64db7)
![Screenshot 2024-11-24 075311](https://github.com/user-attachments/assets/b53274f2-ca0f-40ab-99fa-b1f7c61db218)

**Comment by NexIsDumb** [comment, marked as answer]
dont listen to vsc errors, as i said before theyre not linked to cne itself

**Reply by chronicsilly** [reply]
is it just me or are the images not loading

**Reply by THEkatinamicrowave** [reply]
it's just you homie
although if you could help with that one shader I posted about I would be fine with that

**Reply by chronicsilly** [reply]
> it's just you homie although if you could help with that one shader I posted about I would be fine with that

no

**Reply by r6915ee** [reply]
> is it just me or are the images not loading

It's likely about cached images.

**Reply by chronicsilly** [reply]
> > is it just me or are the images not loading
> 
> It's likely about cached images.

probably my vpn being weird, i turned it off and i can see them now
